mne/tests/test_epochs.py

```diff
@@ -3878,5 +3878,63 @@ def test_epoch_annotations_cases():
     assert 'multiple' in first_epoch_ant
 
     # test that concatenation does not preserve epochs
+    old_epochs = epochs.copy()
     with pytest.warns(RuntimeWarning, match='Annotations'):
         concatenate_epochs([epochs, epochs_one])
+
+    # concatenation should not change the Epochs annotations
+    assert epochs.annotations == old_epochs.annotations
+
+
+def test_epochs_saving_with_annotations(tmp_path):
+    """Test Epochs save correctly with Annotations."""
+    # set up a test dataset
+    data = np.random.RandomState(0).randn(1, 600)
+    sfreq = 100.
+    info = create_info(ch_names=['MEG1'], ch_types=['grad'], sfreq=sfreq)
+    raw = RawArray(data, info)
+    ant_dur = 0.4
+    ants = Annotations(
+        onset=[0.4, 0.5, 1.4, 1.6, 3.0, 5.4],
+        duration=[ant_dur, ant_dur, ant_dur, 0, 2.0, 0],
+        description=['before', 'start', 'stop',
+                     'outside', 'multiple', 'bad_noisy'],
+    )
+    raw.set_annotations(ants)
+
+    events = np.zeros((3, 3), dtype=int)
+    events[:, 0] = [100, 300, 500]
+    epochs = Epochs(raw, events=events, tmin=-0.5, tmax=0.5)
```

Comment:
don't copy-paste this from the other test.  Either set up a (private) helper function that both tests call, or create a pytest fixture and pass it in as a test argument.

Reply:
Done.